src/client/pages/ConnectPayment/components/TrustlyCheckout.test.tsx

```diff
@@ -33,46 +32,42 @@ it('opens trustly modal and renders correct iframe url', async () => {
   // @ts-ignore
   window.location = new URL(location)
 
-  const wrapper = mount(
-    <StaticRouter location={basePath}>
-      <MockedProvider
-        mocks={[
-          {
-            request: {
-              query: REGISTER_DIRECT_DEBIT_MUTATION,
-              variables: {
-                clientContext: {
-                  successUrl: location + '/success',
-                  failureUrl: location + '/fail',
-                },
+  const { getByRole, getByText } = renderComponent(
+    <MockedProvider
+      mocks={[
+        {
+          request: {
+            query: REGISTER_DIRECT_DEBIT_MUTATION,
+            variables: {
+              clientContext: {
+                successUrl: location + '/success',
+                failureUrl: location + '/fail',
               },
             },
-            result: {
-              data: {
-                registerDirectDebit: {
-                  orderId: 'abc123',
-                  url: trustlyUrl,
-                  __typename: 'DirectDebitResponse',
-                },
+          },
+          result: {
+            data: {
+              registerDirectDebit: {
+                orderId: 'abc123',
+                url: trustlyUrl,
+                __typename: 'DirectDebitResponse',
               },
             },
           },
-        ]}
-      >
-        <StaticTextKeyProvider>
-          <TrustlyCheckout />
-        </StaticTextKeyProvider>
-      </MockedProvider>
-    </StaticRouter>,
+        },
+      ]}
+    >
+      <StaticTextKeyProvider>
```

Comment:
So maybe the `StaticTextKeyProvider` is redundant here now that the `StaticTextKeyProvider` is included in the `renderComponent` function? Same on line 65 in `TrustlyModal.test.tsx`. Or am I missing something?

Reply:
Good catch! Removed it :)